daemon/inspect.go

```diff
@@ -32,7 +32,7 @@ func (daemon *Daemon) ContainerInspect(name string) (*types.ContainerJSON, error
 		})
 	}
 
-	return &types.ContainerJSON{base, mountPoints, container.Config}, nil
+	return &types.ContainerJSON{*base, mountPoints, *container.Config}, nil
```

Comment:
it's not like this is the only code where asterisks exist :wink:

I prefer to use values for embedded structs than pointers. It's safer to call methods from the parent this way.